tools/dwt_cor_ava_perclass/execute_dwt_cor_aVa_perClass.xml

```diff
@@ -1,33 +1,214 @@
 <tool id="compute_p-values_correlation_coefficients_feature_occurrences_between_two_datasets_using_discrete_wavelet_transfom" name="Compute P-values and Correlation Coefficients for Feature Occurrences" version="1.0.0">
   <description>between two datasets using Discrete Wavelet Transfoms</description>
-  
-  <command interpreter="perl">
-  	execute_dwt_cor_aVa_perClass.pl $inputFile1 $inputFile2 $outputFile1 $outputFile2
+  <requirements>
+    <requirement type="package" version="1.7.5">r-waveslim</requirement>
+    <requirement type="package" version="4.6.8">r-wavethresh</requirement>
+    <requirement type="package" version="2.4_5">r-rwave</requirement>
+  </requirements>
+  <command detect_errors="exit_code">
+    Rscript --vanilla '$script_file'
   </command>
+  <configfiles>
+    <configfile name="script_file"><![CDATA[
```

Comment:
Thanks for dropping the Perl script! Maybe this can be a standalone R script? It seems easier to maintain, e.g. with syntax highlighting in the code editor.

Similarly for the other dwt tools.

Reply:
apparently I did this already..